ship_builder.html

```diff
@@ -1,6 +1,6 @@
 ---
 ---
-<html>
+<html lang="en">
```

Comment:
`lang="en-US"` might be better, although it's okay as is.

Reply:
```suggestion
<html lang="en-US">
```